lighthouse-core/audits/network-requests.js

```diff
@@ -50,6 +50,10 @@ class NetworkRequests extends Audit {
           statusCode: record.statusCode,
           mimeType: record.mimeType,
           resourceType: record.resourceType,
+          endTimeDeltaMs: record.endTimeDeltaMs, // Only exists on Lightrider runs
```

Comment:
maybe prefix these with `lr` or `lrStats` just to make it extra clear what they are?

Reply:
Done.